src/components/Footer/index.module.css

```diff
@@ -1,119 +1,45 @@
-.blue {
-  padding: 56px 20px 20px;
-
-  background-color: #045cfc;
-  color: #fcfcfc;
-}
-
-.blue .content {
-  display: flex;
-  flex-direction: column;
-  max-width: 1184px;
-  margin-right: auto;
-  margin-left: auto;
-}
-
-.logo {
-  height: 40px;
-  margin-bottom: 20px;
-
-  font-size: 28px;
-}
-
-.callToAction {
-  margin-bottom: 28px;
+.container {
+  display: grid;
+  grid-gap: 84px;
 }
 
-.locations {
+.content {
   display: grid;
   grid-row-gap: 28px;
-  width: 100%;
-  margin-bottom: 28px;
 }
 
-.social {
-  display: flex;
-  flex-direction: column;
-  margin-bottom: 56px;
-}
+.itemLabel {
+  padding-bottom: 16px;
 
-.socialLabel {
-  margin-bottom: 20px;
+  color: #045cfc;
 }
 
-.white {
-  padding: 20px 0;
-
-  background-color: #ffffff;
-}
-
-.white .content {
-  height: auto;
-  max-width: 100%;
-  margin-right: auto;
-  margin-left: auto;
-}
-
-.norte2020Links {
-  list-style: none;
-}
-
-.norte2020Link {
-  text-align: center;
-}
-
-@media (min-width: 651px) {
-  .white .content {
-    max-width: 650px;
-  }
+.links {
+  grid-row: auto auto;
+  display: grid;
+  grid-column-gap: 20px;
+  grid-row-gap: 16px;
+  grid-template-columns: 168px auto;
 }
 
 @media (min-width: 768px) {
-  .blue .content {
-    align-items: center;
-  }
-
-  .locations {
-    grid-column-gap: 20px;
-    grid-row-gap: initial;
+  .content {
+    grid-column-gap: 28px;
+    grid-row-gap: 56px;
     grid-template-columns: 1fr 1fr;
-    grid-template-rows: initial;
-  }
-
-  .social {
-    flex-direction: row;
-  }
-
-  .socialLabel {
-    margin-right: 20px;
-    margin-bottom: 0;
   }
 }
 
 @media (min-width: 950px) {
-  .blue {
-    padding: 112px 28px 28px;
-  }
-
-  .logo {
-    max-width: 176px;
-    margin-bottom: 28px;
-  }
-
-  .callToAction {
-    margin-bottom: 56px;
-  }
-
-  .locations {
-    grid-column-gap: 28px;
-    margin-bottom: 56px;
+  .container {
+    grid-gap: 112px;
   }
 
-  .social {
-    margin-bottom: 112px;
+  .content {
+    margin: 0 auto;
```

Comment:
This is what development looks like at `1440px`:

<img width="883" alt="Screenshot 2023-03-28 at 10 24 04" src="https://user-images.githubusercontent.com/26620750/228191729-a0312263-08f5-456f-9875-16bd63b8d1b3.png">

This is what the design looks like at `1440px`

<img width="777" alt="Screenshot 2023-03-28 at 10 25 12" src="https://user-images.githubusercontent.com/26620750/228191923-7eaf97c5-992b-4803-a15b-130ffa924c43.png">